### PR TITLE
Make pycrypto optional.

### PR DESCRIPTION
**Pull request by j-bennet**
This handles issue #169 about making pycrypto dependency optional.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -29,8 +29,10 @@
             'PyMySQL >= 0.6.2',
             'sqlparse >= 0.1.16',
             'configobj >= 5.0.6',
-            'pycrypto >= 2.6.1',
             ],
+        extras_require={
+            'parse_mylogin_cnf': ['pycrypto >= 2.6.1']
+        },
```
Comment (amjith) [inline setup.py]: This makes it so pycrypto is never installed by default. It is only installed when someone goes out and does `pip install pycrypto`. 

I'd prefer to take the approach of not installing pycrypto in Windows but install it everywhere else. 

See https://github.com/dbcli/pgcli/pull/397 for reference on how to do this.

**Comment by amjith** [comment]
I've checked the remaining changes and they work great. Good job!

**Comment by j-bennet** [comment]
@amjith Didn't see this approach before - thanks! I hope it's good now.

**Comment by amjith** [comment]
Looks good to me. :+1: 

Thanks for the update. 

:soccer:

**Pull request merged**